### DISCUSSION TITLE
Export Audiobooks?

### DISCUSSION THREAD

**Discussion by BajrangB** [TortugaPower/BookPlayer#581] [Q&A]
Hi, I was wondering if there was a way to export books that have already been imported into Bookplayer?

I tried using itunes but I see only randomly named files in the ‘Processed’ folder. It would be great if this was possible because I deleted the original zipped files of some rare  books I had imported into Bookplayer. Thanks!

**Comment by GianniCarlo** [comment]
Hi @BajrangB , you should be able to export them via the `Options` menu, which can be accessed two ways, the first one is just swiping left on the book in the library, and the `Options` button will be revealed. The second one is to press the `Edit` button in the top right corner, selecting the book, and tapping the `...` button in the bottom floating bar, which will reveal the same options.

From there, you can press the `Share` option, and it will open up the Share sheet from iOS, from there you can airdop it, or move it with any of your installed apps that may support receiving that app. There's only one caveat though, we only support this for one book at a time, if you have multiple books selected at once, then the `Share` option won't be available

**Reply by BajrangB** [reply]
I tried your suggestion and even though only one book was selected, the SHARE button was greyed out...could it be because it has multiple files in it? In that case, is there a way to export the book? Thanks!

P.S. I'm grateful for all the hard work that goes into this project! Thanks to all the contributors!

**Reply by GianniCarlo** [reply]
Ah yes, you're correct, the item selected is actually considered a playlist, due to memory concerns when exporting, we limited the quantity of items to be exported to just 1 book. This may in one way be solved by https://github.com/TortugaPower/BookPlayer/issues/389 , but we're still some time away from completing that feature. In the meantime, the only way would be to go inside the playlist and export each individual file one at a time :/

**Reply by BajrangB** [reply]
Ok, thanks! Will wait for the update. As they say, patience is a virtue! :)
